src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -277,23 +295,23 @@ public final Builder name(String name) {
 		 * Set the options to use for configuring {@link ConnectionProvider} acquire timeout.
 		 * Default to DEFAULT_POOL_ACQUIRE_TIMEOUT.
```

Comment:
This could be a `{@link ...}` to the constant.

Reply:
done